tests/back-end/unit/lib/hooks/logger.ts

```diff
@@ -35,13 +35,13 @@ function makeStubRequest(onLog: (kind: LogFunctionKind, msg: string, data?: obje
   };
 }
 
-describe('loggerHook', () => {
+describe('loggerHook', function() {
```

Comment:
Great job finding tooling to enforce [no arrow functions per Mocha guidance](https://mochajs.org/#arrow-functions)!